specification/compute/Compute.Management/models.tsp

```diff
@@ -1830,6 +1906,25 @@ model EventGridAndResourceGraph {
    * Specifies if event grid and resource graph is enabled for Scheduled event related configurations.
    */
   enable?: boolean;
+
+  /**
+   * Specifies the api-version to determine which Scheduled Events configuration schema version will be delivered.
+   */
+  @added(Versions.v2025_04_01)
+  scheduledEventsApiVersion?: string;
```

Comment:
Could you explain this a bit more ? Is the response schema changing based on the api-version sent here ? 

How can end users expect which schema to be sent to them ?